pkg/webhooks/regularuser/regularuser.go

```diff
@@ -181,6 +182,13 @@ func (s *RegularuserWebhook) authorized(request admissionctl.Request) admissionc
 		ret.UID = request.AdmissionRequest.UID
 		return ret
 	}
+	for _, user := range adminUsers {
+		if request.AdmissionRequest.UserInfo.Username == user {
```

Comment:
`if utils.SliceContains(request.AdmissionRequest.UserInfo.Username, adminUsers` suitable here?

Reply:
oh, yeah it would :)